lib/src/constants.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:math';
+import 'dart:ui';
+
+class Constants {
+  Constants._();
+
+  static final Random _random = Random();
+  static final int _maxColor = 256;
+
+  /// minimum and maximum dates are 100,000,000 days before and after epochDate
+  ///
+  static final DateTime epochDate = DateTime(1970, 1, 1);
+  static final DateTime maxDate = DateTime(275760, 1, 1);
```

Comment:
Make them `const`

Reply:
Can not assign DateTime object as constant.

Tried using,

`static const DateTime epochDate = const DateTime(1970, 1,1);
 static const DateTime maxDate = DateTime(275760, 1,1)`

none of them are working.